### PR TITLE
[Remove Blocking usages]LPS-186857 document-library-service | DLSubscriptionSender

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by W2jx** [comment]
ci:test:startup

**Comment by W2jx** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:document

**Comment by W2jx** [comment]
no conflicts(6/23 14.49)

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLSubscriptionSender.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ protected Boolean hasSubscribePermission(
 			ServiceProxyFactory.newServiceTrackedInstance(
 				ModelResourcePermission.class, DLSubscriptionSender.class,
 				"_dlFolderModelResourcePermission",
-				"(model.class.name=" + DLFolder.class.getName() + ")", true);
+				"(model.class.name=" + DLFolder.class.getName() + ")", false);
```
Comment (dantewang) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/util/DLSubscriptionSender.java:63]: @W2jx This class is in module, and it's only used by one component. Normally we can reference the model resource permission and pass via constructor. However, considering SubscriptionSender may be serialized, please try to use `Snapshot` instead.

**Review by dantewang — CHANGES_REQUESTED** [review]
cc @JanisZhang @LisztLiszt @slnn

**Pull request closed without merging**